jpf-symbc/bin/jpf-sv-comp.sh

```diff
@@ -63,12 +63,12 @@ echo "symbolic.arrays=true" >> $DIR/config.jpf
 echo "listener = .symbc.SymbolicListener" >> $DIR/config.jpf
 
 # run SPF
-export DYLD_LIBRARY_PATH=`pwd`/jpf-symbc/lib:$DYLD_LIBRARY_PATH
+export LD_LIBRARY_PATH=`pwd`/jpf-symbc/lib:$LD_LIBRARY_PATH
 #jpf-core/bin/jpf $DIR/config.jpf
 if test -z "$JVM_FLAGS"; then
   JVM_FLAGS="-Xmx1024m -ea"
 fi
-timeout 900 /usr/lib/jvm/java-8-openjdk-amd64/bin/java $JVM_FLAGS -jar `pwd`/jpf-core/build/RunJPF.jar $DIR/config.jpf | tee $LOG
+timeout 900 /Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/bin/java $JVM_FLAGS -jar `pwd`/jpf-core/build/RunJPF.jar $DIR/config.jpf | tee $LOG
```

Comment:
Same why have you changed this path?

Reply:
Oh it's my mistake. I changed this to test on my machine..I fixed this. Thank you!